src/azure-cli/azure/cli/command_modules/role/tests/latest/test_role_commands_thru_mock.py

```diff
@@ -11,16 +11,14 @@
 
 from azure.cli.core.mock import DummyCli
 
-from azure.mgmt.authorization.models import RoleDefinition, RoleAssignmentCreateParameters
+from azure.mgmt.authorization.models import RoleDefinition
 from azure.graphrbac.models import (Application, ServicePrincipal, GraphErrorException,
-                                    ApplicationUpdateParameters, GetObjectsParameters)
+                                    ApplicationUpdateParameters)
 from azure.cli.command_modules.role.custom import (create_role_definition,
                                                    update_role_definition,
                                                    create_service_principal_for_rbac,
                                                    reset_service_principal_credential,
-                                                   update_application, _try_x509_pem,
```

Comment:
I need to investigate where these changes came from... not something I did manually.

Reply:
Actually, digging into it, it seems like these are things the PyCharm cleaned up for me because they were unused. I'm going to go ahead and assume these changes can stay. (Also low-pri because these are just test package imports.)